### PR TITLE
Add patient details upon click

### PR DESCRIPTION
**Pull request by elineelorm**
> Displayed patient details using overlays

> I hope to add more information based on what questions we come up with for the questionnaire.

### CODE REVIEW

**Comment by elineelorm** [comment]
Also, the "add patient" button doesn't do anything for now

File: app/src/main/webapp/patients.html [hunk 1]
```diff
@@ -15,54 +15,94 @@ <h2 class="patient-title" id="list-heading">
...
-                     on June 28, 2021 
-                </time>
-                <h3>
-                    Gina Jackson
-                </h3>
-                <h5> 
-                    Age: 15
-                </h5>
-            </patient>
+            <button onclick="on1()">
```
Comment (sbahaud) [inline app/src/main/webapp/patients.html:18]: Having a separate function for each patient, would not scale when you have a lot of patients. The reason I recommended that was so you test and see if that works.

This doc file has some examples from my previous team that added a list of items that were clickable. Do take a look - https://docs.google.com/document/d/1FSJyjlTSLoeg0V7OaOPHCbvAE880BCpu2FICIo43YFM/edit

I am ok to approve this for now, while you continue to explore your options

File: app/src/main/webapp/patients.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+function on1() {
+  document.getElementById("overlay1").style.display = "block";
```
Comment (sbahaud) [inline app/src/main/webapp/patients.js:2]: Can you try having a single function and passing in a different parameter for each patient?

**Review by sbahaud — COMMENTED** [review]
Left some comments

File: app/src/main/webapp/patients.html (hunk 1 above)
Comment (elineelorm) [reply app/src/main/webapp/patients.html:18]: okay, will do

File: app/src/main/webapp/patients.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Function to display the hidden list content
+ * @param {*} tabName 
+ */
+function openTab(tabName) {
+  var i, x;
+  x = document.getElementsByClassName("containerTab");
```
Comment (sbahaud) [inline app/src/main/webapp/patients.js:7]: Instead of x, use a variable name that is more appropriate. container maybe?

**Review by sbahaud — APPROVED** [review]
Approved with minor suggestion

**Pull request merged**